core/src/main/java/org/elasticsearch/action/admin/indices/flush/ShardFlushRequest.java

```diff
@@ -53,5 +54,8 @@ public void writeTo(StreamOutput out) throws IOException {
         request.writeTo(out);
     }
 
-
+    @Override
+    public String toString() {
+        return "flush {" + shardId() + "}";
```

Comment:
I added a simple toString method to ReplicationRequest in https://github.com/areek/elasticsearch/commit/6fa8140adf3b9c774431b5f815543930d144cde2 for providing the index and shard info if resolved or just the index name if the request still needs resolving. Note that I intentionally didn't use this for index and delete requests.